bin/watch-dev.js

```diff
@@ -21,19 +21,20 @@ async function build() {
     watch: !once,
     babelConfig: {
       plugins: [runtime, styled],
-      runtimeHelpers: true,
+      babelHelpers: 'runtime',
       include: [
-        `${__dirname}/../src/frontend/**`,
-        `${__dirname}/../src/locale/*`,
-        `${__dirname}/../src/utils/**`,
-        `${__dirname}/../src/backend/utils/view-helpers/view-helpers.ts`,
-        `${__dirname}/../src/backend/utils/filter/filter.ts`,
-        `${__dirname}/../src/backend/decorators/**`,
+        'src/frontend/**',
+        'src/locale/*',
+        'src/utils/**',
+        'src/backend/utils/view-helpers/view-helpers.ts',
+        'src/backend/utils/filter/filter.ts',
+        'src/backend/decorators/**',
```

Comment:
Probably new versions of `@rollup/plugin-babel` resolves `include` paths in a different way